Automattic/jetpack
# wpcom Features: add new Holiday Snow feature

## Proposed changes:

This PR brings a new feature to the mu-wpcom package:

- That feature can be toggled via Settings > General in wp-admin.
- It is active from December 1st to January 4th by default.

<img width="637" alt="image" src="https://github.com/user-attachments/assets/a16f3a94-b993-4cae-b426-84283c57ae7c">


### Other information:


## Jetpack product discussion

https://github.com/Automattic/dotcom-forge/issues/10082

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

> [!NOTE]
> This can be tested on WordPress.com (Simple and WoA)
> When testing on WoA, add the following to your site's `wp-config.php` file:
> ```php
> define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );
> define( 'JETPACK_AUTOLOAD_DEV', true );
> ```

1. After adding the branch to your test site,
2. Go to Settings > General (`:site:/wp-admin/options-general.php`)
3. Toggle the feature on (it should be off by default)
4. Visit the site's frontend
    * You should see snow falling
5. Toggle the feature off and save your changes
    * The snow should be gone.

If you do not use the classic interface for that site in your WordPress.com settings, you can test in calypso using this Calypso PR:
https://github.com/Automattic/wp-calypso/pull/97127
